gm4_blast_furnaces/data/blast_furnaces/functions/search_furnace.mcfunction

```diff
@@ -0,0 +1,9 @@
+# Called from handle_block_structure.mcfunction
+
+# @s = @e[tag=gm4_bf_output, tag=!gm4_bf_has_furnace]
+
+
+execute if block ~1 ~ ~ furnace[facing=east] run function blast_furnaces:furnace_tagging/east
+execute if block ~-1 ~ ~ furnace[facing=west] run function blast_furnaces:furnace_tagging/west
+execute if block ~ ~ ~1 furnace[facing=south] run function blast_furnaces:furnace_tagging/south
+execute if block ~ ~ ~-1 furnace[facing=north] run function blast_furnaces:furnace_tagging/north
```

Comment:
It might be better here to `execute store success` to a fake player, then if assign the `has_furnace` tag based upon that. This would eliminate the need for 4 functions.

Reply:
Could you elaborate on that?